### PR TITLE
[QA-408]: Added scripts for AIS scenarios

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Added scripts for AIS scenarios

#### Changes:
- Added permissions for `PerformanceTester` role in CF template.
- Added `Persist Intervention` scenario.
- Added `Retrieve Intervention` scenario

---

### Why?
To conduct AIS performance testing

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -40,9 +40,12 @@ Mappings:
   ContraIndicators:
     AWS:
       AccountID: "755415363251"
-  TxMA:
+  TxMABuild:
     AWS:
       AccountID: "750703655225"
+  TxMAStaging:
+    AWS:
+      AccountID: "178023842775"
```
Comment (Tom-Dann) [inline deploy/template.yaml:48]:
```suggestion
  TxMA:
    Build:
      AccountID: "750703655225"
    Staging:
      AccountID: "178023842775"
```

File: deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export interface TICF_ACCOUNT_INTERVENTION {
```
Comment (Tom-Dann) [inline deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts:1]:
```suggestion
export interface TicfAccountIntervention {
```
Prefer PascalCase for types/interfaces

File: deploy/scripts/src/ais/test.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      exec: 'persistIV'
+    },
+    retrieveIV: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 11400,
+      stages: [
+        { target: 3800, duration: '15m' }, // Ramp up to 3800 iterations per second in 15 minutes
```
Comment (Tom-Dann) [inline deploy/scripts/src/ais/test.ts:88]: Should this be 1900 and two calls in the scenario?

File: deploy/template.yaml (hunk 1 above)
Comment (abhinaysrivastavaperf) [reply deploy/template.yaml:48]: Done

File: deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts (hunk 2 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts:1]: Done

File: deploy/scripts/src/ais/test.ts (hunk 3 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ais/test.ts:88]: No the worst case scenario would be the peak volume of calls from Auth (1900/s) and Core (1900/s) simultaneously. The GET call from both Auth and Core is same so testing this at 3800/s would be the worst case peak volume of this GET call.

File: deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts
Comment (Tom-Dann) [inline deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts:1]: Makes sense to move this interface definition to the same file as `deploy/scripts/src/ais/requestGenerator/aisReqGen.ts`

**Review by Tom-Dann — APPROVED** [review]

**Pull request merged**